src/background/redux/root-reducer.ts

```diff
@@ -21,7 +22,8 @@ const rootReducer = combineReducers({
   loginRetryLockoutTime,
   lastActivityTime,
   activeOrigin,
-  settings
+  settings,
+  recipientPublicKeys
```

Comment:
@ost-ptk what is the rationale for putting `recipientPublicKeys` into the global state?
it's a form state and should be declared on the form container level
please remove from redux completely

Reply:
Is this a mechanism to store recently used public keys? if yes then it's named inappropriately, should be named recentRecipientPublicKeyList

Reply:
done